alphadia/workflow/peptidecentric.py

```diff
@@ -323,32 +323,51 @@ def get_ordered_optimizers(self):
         else:
             mobility_optimizer = None
 
-        optimizers = [
-            ms2_optimizer,
-            rt_optimizer,
-            ms1_optimizer,
-            mobility_optimizer,
-        ]
-        targeted_optimizers = [
-            [
-                optimizer
+        if self.config["optimization"]["order_of_optimization"] is None:
```

Comment:
the `if` branch could be removed if the `default.yaml` provided a default instead of `None`

Reply:
It couldn't be removed while maintaining precisely the same functionality, which is to ensure that targeted optimizers always come before automatic optimizers. Whether this functionality is worth maintaining is a separate question, but I thought it best to leave it in place for the time being as it seems a reasonable default.

Reply:
I see .. should the "targeted optimizers always come before automatic optimizers." be added as a comment to the yaml?

Reply:
OK.